temporalio/client.py

```diff
@@ -0,0 +1,935 @@
+"""Client for accessing Temporal."""
+
+import logging
+import os
+import socket
+import uuid
+from dataclasses import dataclass
+from datetime import timedelta
+from enum import IntEnum
+from typing import Any, Generic, Iterable, Mapping, Optional, TypeVar, Union, cast
+
+import temporalio.api.common.v1
+import temporalio.api.enums.v1
+import temporalio.api.failure.v1
+import temporalio.api.history.v1
+import temporalio.api.taskqueue.v1
+import temporalio.api.workflowservice.v1
+import temporalio.common
+import temporalio.converter
+import temporalio.failure
+import temporalio.workflow_service
+
+logger = logging.getLogger(__name__)
+
+
+class WorkflowIDReusePolicy(IntEnum):
```

Comment:
This should to into `common`, you'll use this type to start child workflows too.

Reply:
Done